services/task-svc/internal/patient/patient.go

```diff
@@ -0,0 +1,101 @@
+package patient
+
+import (
+	"context"
+	"github.com/google/uuid"
+	zlog "github.com/rs/zerolog/log"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+	"hwgorm"
+	"task-svc/api"
+)
+
+type Base struct {
+	HumanReadableIdentifier string `gorm:"column:human_readable_identifier"`
```

Comment:
is `gorm:column` really needed here (and below)?

Reply:
We should be explicit.

Reply:
[GORM Docs:](https://gorm.io/docs/models.html#Conventions)
> GORM prefers convention over configuration. [...] If you follow the conventions adopted by GORM, you’ll need to write very little configuration/code.

I personally don't really care if we are explicit or not, as long as we are consistent. So we should either add the gorm tags to all existing models or remove them here.